browser/notifications/notification_platform_bridge_brave_custom_notification.cc

```diff
@@ -0,0 +1,178 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/notifications/notification_platform_bridge_brave_custom_notification.h"
+
+#include <memory>
+#include <string>
+
+#include "base/bind.h"
+#include "base/callback.h"
+#include "base/memory/scoped_refptr.h"
+#include "base/run_loop.h"
+#include "base/task/post_task.h"
+#include "brave/components/brave_ads/browser/ads_notification_handler.h"
```

Comment:
this is definitely a problem. If this is a generic custom notification it should not have dependencies on brave_ads. If it is specific to ads then it should be in browser/brave_ads/notifications